extensions/wikia/ThemeDesigner/js/ThemeDesigner.js

```diff
@@ -176,7 +176,7 @@
 
 					ThemeDesigner.set('theme', targetObject.attr('data-theme'));
 					if (ThemeDesigner.checkBgImageIsSet()) {
-						ThemeDesigner.loadImage(ThemeDesigner.settings['background-image']);
```

Comment:
This is to assign properties properly to `background-image-width` and friends rather than `undefined-width`, etc. This is a general fix that applies to editing theme settings.